### DISCUSSION TITLE
Why does nobody have an Application level component in their UI suite?

### DISCUSSION THREAD

**Discussion by resistdesign** [chakra-ui/chakra-ui#2949] [Ideas] [answered]
Why are there no Application and overall screen level layout components?

**Comment by chr-ge** [comment, marked as answer]
This is a requested feature. You can take a look at these issues: https://github.com/chakra-ui/chakra-ui/issues/1281 and https://github.com/chakra-ui/chakra-ui/issues/2940

**Reply by with-heart** [reply]
To add a little context to the above, Chakra is focused on providing a low-level set of accessible components that can be composed to achieve any outcome rather than looking to provide literally any component anyone thinks of. That's why the above issues are more focused on adding examples of these types of layouts on the site rather than adding them to Chakra itself.
